internal/pkg/term/selector/selector.go

```diff
@@ -133,6 +160,55 @@ func (s *DeployedService) String() string {
 	return fmt.Sprintf("%s (%s)", s.Svc, s.Env)
 }
 
+// VPC has the user select an available VPC.
+func (s *Ec2Select) VPC(prompt, help string) (string, error) {
+	vpcIDs, err := s.ec2Svc.ListVPC()
+	if err != nil {
+		return "", fmt.Errorf("list VPC ID: %w", err)
+	}
+	if len(vpcIDs) == 0 {
+		return "", ErrVpcNotFound
+	}
+	vpcID, err := s.prompt.SelectOne(
+		prompt, help,
+		vpcIDs)
+	if err != nil {
+		return "", fmt.Errorf("select VPC: %w", err)
+	}
+	return vpcID, nil
+}
+
+// PublicSubnet has the user select public subnets given the VPC ID.
+func (s *Ec2Select) PublicSubnet(prompt, help, vpcID string) ([]string, error) {
```

Comment:
nit: `PublicSubnets` can we mention that it allows to choose multiple subnets in the comments too